models/index.js

```diff
@@ -15,6 +33,48 @@ dbConfig.logging = function(msg, timeMs) {
   log.debug("%s [%d ms]", msg, timeMs);
 };
 
+// String-based operators are deprecated in sequelize v4 as a security concern.
+// http://docs.sequelizejs.com/manual/tutorial/querying.html#operators-security
+// Because they are currently used via the API, we need to keep them enabled.
+// The following definition explicitly enables all supported aliases. 
+const Op = Sequelize.Op;
+dbConfig.operatorsAliases = {
+  $eq: Op.eq,
+  $ne: Op.ne,
+  $gte: Op.gte,
+  $gt: Op.gt,
+  $lte: Op.lte,
+  $lt: Op.lt,
+  $not: Op.not,
+  $in: Op.in,
+  $notIn: Op.notIn,
+  $is: Op.is,
+  $like: Op.like,
+  $notLike: Op.notLike,
+  $iLike: Op.iLike,
+  $notILike: Op.notILike,
+  $regexp: Op.regexp,
+  $notRegexp: Op.notRegexp,
+  $iRegexp: Op.iRegexp,
+  $notIRegexp: Op.notIRegexp,
+  $between: Op.between,
+  $notBetween: Op.notBetween,
+  $overlap: Op.overlap,
+  $contains: Op.contains,
+  $contained: Op.contained,
+  $adjacent: Op.adjacent,
+  $strictLeft: Op.strictLeft,
+  $strictRight: Op.strictRight,
+  $noExtendRight: Op.noExtendRight,
+  $noExtendLeft: Op.noExtendLeft,
+  $and: Op.and,
+  $or: Op.or,
+  $any: Op.any,
+  $all: Op.all,
+  $values: Op.values,
+  $col: Op.col
+};
```

Comment:
Lets lock down what's possible. The following can be left out there:

* $regexp
* $notRegexp
* $iRegexp
* $notIRegexp
* $overlap
* $contained
* $adjacent
* $strictLeft
* $strictRight
* $noExtendRight
* $noExtendLeft
* $values
* $col